nixos/modules/services/misc/etebase-server.nix

```diff
@@ -46,6 +46,20 @@ in
         '';
       };
 
+      package = mkOption {
+        type = types.package;
+        default = cfg.pythonPackage.pkgs.etebase-server;
+        defaultText = literalExpression "pkgs.etebase-server";
+        description = lib.mdDoc "etebase-server package to use.";
+      };
+
+      pythonPackage = mkOption {
```

Comment:
We should implement this by passing thru the interpreter used to build the package. Maybe that is already available at .python, not fully sure.

Reply:
Happy to do this, but wouldn't using "the interpreter used to build the package" imply infinite recursion?

The default for package is cfg.pythonPackages.pkgs.etebase-server, so I can't really evaluate that to access `.pythonModule` before I know the python interpreter to use?

Reply:
You should be able to access cfg.package.python without triggering an infinite recursion as this is just passed thru, too.